CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -916,7 +1024,7 @@ local function isFreeItem()
 end
 
 local getPlayerBalance
-if useNewMarketplaceMethods() then
+if false then --useNewMarketplaceMethods() then
```

Comment:
Oops again. I commented that out to check without mock purchases without changing the C++ side so I didn't actually spend my hard-earned R$. I think this is the last place that I changed for that, but I'll give it a look-over again.